homeassistant/components/flick_electric/config_flow.py

```diff
@@ -86,13 +86,16 @@ async def async_step_select_account(
             except (APIException, ClientResponseError):
                 errors["base"] = "cannot_connect"
             except AuthException:
-                # Send user back to enter credentials
-                return await self.async_step_user({**self.data})
+                # We should never get here as we have a valid token
+                return self.async_abort(reason="unknown")
```

Comment:
Let's abort with a reason that explains that we don't have access for this particular account and that they should contact Flick